fix engravings that have gone out of sync (#1099)
DFHack/scripts: docs/fix/engravings.rst

fix/engravings

> =============

Author: ```suggestion
==============
```

.. dfhack-tool::
    :summary: Fixes unengravable corrupted tiles so they are able to be engraved.
    :tags: fort bugfix

> When placing a new wall or new floor down where a previous engraved tiletype was, the tile may be corrupted and unengravable. 
> This fix removes corrupted engravings from those tiles automatically so those tiletypes may be engraved again.

Author: ```suggestion
When constructing a new wall or new floor over a previously engraved tile, the tile may become corrupted and unengravable. 
This fix detects the problem and resets the state of those tiles so they may be engraved again.

```

Usage
-----

::

>     fix/engravings

Author: ```suggestion
    fix/engravings [<options>]

Options
-------

``-q``, ``--quiet``
    Only output status when something was actually fixed.
```
